code/controllers/subsystems/job.dm

```diff
@@ -113,6 +113,8 @@
 			position_limit = job.get_spawn_positions()
 		if((job.current_positions < position_limit) || position_limit == -1)
 			Debug("Player: [player] is now Rank: [rank], JCP:[job.current_positions], JPL:[position_limit]")
```

Comment:
The time played should be a configuration value.
It should also be possible to disable this system via a special configuration value. i.e. -1